### DISCUSSION TITLE
AppComposer canvas

### DISCUSSION THREAD

**Discussion by atali** [cloudscape-design/components#943] [Q&A] [answered]
Do you plan to implement a canva component like the use in AppComposer ? or is there a way to do it with the composition of the current components already available in cloudscape ?

#944

**Comment by timogasda** [comment]
Could you please describe in a little more detail what an "AppComposer canvas" is and how it works?

**Comment by atali** [comment]
Hi @timogasda 

I am referring to the canvas component used by [App composer ](https://aws.amazon.com/application-composer/)

I would like to be able to use the same kind of component to drag and drop components on top of the canva and link them together. When a user interacts with a component, I would expect some events about what he is doing in order to edit/move/delete the component itself.

Do you think it's something you can support ?

See below an example:

![AppComposer](https://a.b.cdn.console.awsstatic.com/a/v1/FKJNWCP7VKI52YSCJCS7JK5QQD2LOFGPHUEKA4ABXNEFS7VM3PSQ/images/home-demo-light.gif)

**Comment by just-boris** [comment, marked as answer]
No, we have no plans of building such component
